src/Component/Symbolizer/Editor/Editor.example.md

```diff
@@ -0,0 +1,44 @@
+This demonstrates the use of Editor.
+
+```jsx
+const React = require('react');
+const { Editor } = require('../../../index');
```

Comment:
`index.js` is the actual entrypoint if you use geostyler as a dependency without treeshaking. This way, users can see which and how components are being exposed. Btw, react-geo is importing its components in the same way

Reply:
I know it is an entrypoint, but as far as I have understood it, it is one we would never use and we would not usually recommend.

If the above is correct, we should hint at this in our examples. But as I said, this is non-blocking of course

Reply:
But this would be worse, wouldn't it. do we want people to import in a way which makes treeshaking hard/impossible? I probably don't understand the nitty gritty details...

We can merge and discuss afterwards

Reply:
Here is the magic in react-geo: https://github.com/terrestris/react-geo/blob/master/styleguide.config.js#L22:L25